From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@sourcemage.org>
Date: Sat, 26 Sep 2026 19:46:29 +0200
Subject: [PATCH] xslt: Cast the compiled style to an element pointer

info is an xsltStylePreCompPtr, but xsltCallTemplate expects an
xsltElemPreCompPtr, which triggers -Wincompatible-pointer-types.

Upstream-Status: Pending
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 libgda-xslt/sql_backend.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgda-xslt/sql_backend.c b/libgda-xslt/sql_backend.c
index 74c5308533e6..97b1660b9870 100644
--- a/libgda-xslt/sql_backend.c
+++ b/libgda-xslt/sql_backend.c
@@ -152,7 +152,7 @@ _gda_xslt_bk_section (GdaXsltExCont * exec, GdaXsltIntCont * pdata,
 					if (info != NULL) {
 						xsltCallTemplate
 							(ctxt, node,
-							 cur_node, info);
+							 cur_node, (xsltElemPreCompPtr) info);
 					}
 					else {
 						printf ("the xsltStylePreCompPtr is empthy\n");
